datastream-server/src/main/java/com/linkedin/datastream/server/EventProducer.java

```diff
@@ -90,11 +96,17 @@
   private final boolean _warnLogLatencyEnabled;
   // Latency threshold at which to log a warning message
   private final long _warnLogLatencyThresholdMs;
+  // Whether to enable logging the list of TopicPartitions with events outside alternative SLA
```

Comment:
nit: alternative --> alternate everywhere.

Reply:
good catch 🤣somehow my brain deduped the two words into one